### ISSUE TITLE
Typos, some significantly affecting performance

### ISSUE THREAD

**Issue by e841018** [Hzzone/pytorch-openpose#82]
I attempted to slightly modify the body estimation part in order to fine-tune the backbone network, and noticed several typos. Eventually, I refactored almost everything and noticed significant improvements on estimation accuracy, although using the same pretrained model and thresholds. Since I modified some APIs, and I can't recall what changes I made was critical to the improvement, instead of making a pull request I would like to ask the original contributors to take a look at my version and see if something can help in improving things here.

| Original output for ski.jpg | [My version](https://github.com/e841018/pytorch-openpose) |
| ---- | ---- |
| ![](https://github.com/Hzzone/pytorch-openpose/assets/18109942/ec73739d-4ffd-494f-8a8b-503cef1ff602) | ![](https://github.com/Hzzone/pytorch-openpose/assets/18109942/f0307213-7dcb-406d-8b67-4c94197cfc64) |

Some differences:
* The lower legs of the left-most person is not crossed.
* The second person from the left has lower legs recognized.
* The third person from the left has eyes recognized more accurately.
* The false positive on the upper right is gone.

Some of the changes I remember are listed below, but I think they are not the main reason of the improvement:
* Reverse the RGB order in `util.draw_bodypose` to align with other implementations better.
* Exchange some X and Y in `util.draw_bodypose`. The original version is not erroneous but just misleading.
* Remove an extra ReLU by correcting `_L1` to `_L2` at the end of [this line](https://github.com/Hzzone/pytorch-openpose/blob/5ee71dc10020403dc3def2bb68f9b77c40337ae2/src/model.py#L32)
* Change the clustering behavior [here](https://github.com/Hzzone/pytorch-openpose/blob/5ee71dc10020403dc3def2bb68f9b77c40337ae2/src/body.py#L158).

**Comment by e841018** [comment]
I just found the main source of improvement. My default scale search option is `[0.7, 1.0, 1.3]`, while the original [`scale_search`](https://github.com/Hzzone/pytorch-openpose/blob/5ee71dc10020403dc3def2bb68f9b77c40337ae2/src/body.py#L25) is `[0.5]`. Sorry for the bothering, but I think correcting the typos would still help the readers in the long term, so I would keep the issue open.
